libs/blocks/caas/caas.css

```diff
@@ -527,3 +528,46 @@ dt.title.is-expanded button {
   align-self: baseline;
   margin-top: 6px;
 }
+
+
+/* collapsible panel */
+button.collapse-panel {
+  border: none;
+  font-size: 18px;
+  width: 26px;
+  height: 32px;
+  cursor: pointer;
+}
+
+button.collapse-panel:hover {
+background:#666;
```

Comment:
Space should be left between property and its value

Reply:
done.